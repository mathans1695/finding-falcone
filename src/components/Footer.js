import React, { Component } from 'react';
import '../styles/Footer.css';

class Footer extends Component {
	render() {
		return (
			<div className='Navbar'>
			</div>
		)
	}
}

export default Footer;