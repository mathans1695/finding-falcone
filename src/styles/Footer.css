.Footer {
	font-size: 18px;
	flex-basis: 10%;
	width: 100%;
	text-align: center;
	margin: 20px;
}