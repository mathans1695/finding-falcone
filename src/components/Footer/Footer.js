import React from 'react';
import './Footer.css';

const Footer = () => {
	return (
		<div className='Footer'>
			Coding problem -&nbsp;<a href='http://www.geektrust.in/finding-falcone' target='_blank' rel='noreferrer'>www.geektrust.in/finding-falcone</a>
		</div>
	)
}

export default Footer;