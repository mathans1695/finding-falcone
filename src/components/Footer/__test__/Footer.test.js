import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Footer from '../Footer';

it('Renders correctly', () => {
	const wrapper = shallow(<Footer />);
	
	expect(toJson(wrapper)).toMatchSnapshot();
})