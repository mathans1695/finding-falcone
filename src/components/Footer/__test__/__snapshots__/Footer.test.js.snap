// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="Footer"
>
  Coding problem -Â 
  <a
    href="http://www.geektrust.in/finding-falcone"
    rel="noreferrer"
    target="_blank"
  >
    www.geektrust.in/finding-falcone
  </a>
</div>
`;
